### PR TITLE
fixed versions and added verbose logs

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

**Comment by sbouchet** [comment]
hide whitespaces for better review..

**Review by jeffmaury — COMMENTED** [review]
For me:
- maintenance version is 4.19.0.Final / 12.19.0.GA
- master version is 4.19.1.Final / 12.19.1.GA

File: constants.json [hunk 1]
```diff
@@ -1,98 +1,92 @@
 {
-    "jbide_master_versions": {
-	"function": "listVersions",
-	"project": "JBIDE",
-	"pattern": "4.5.(?!x).*"
+	"jbide_master_versions": {
+		"function": "listVersions",
+		"project": "JBIDE",
+		"pattern": "4.19.1.(?!x).*"
```
Comment (jeffmaury) [inline constants.json]: Not sure to understand the pattern syntax but I would expect that the .1 would be replacing the .(?!x) ?

**Review by jeffmaury — APPROVED** [review]
LGTM

File: constants.json (hunk 1 above)
Comment (sbouchet) [reply constants.json]: pattern is to exclude any .x versions. should works without it, too.

**Pull request merged**